docs/make.jl

```diff
@@ -8,98 +8,98 @@ using Documenter, AbstractAlgebra
 
 makedocs(
          format = Documenter.HTML(;
-            size_threshold_warn = 204800,
-            size_threshold = 409600,
-         ),
+                                  size_threshold_warn = 204800,
```

Comment:
This is what I mean. I messes again with carefully introduced line breaks.

(Sorry, I did not get the indentation right in my github comment.)